### PR TITLE
Add toggle button

### PR DESCRIPTION
**Pull request by jakekinsella**

### CODE REVIEW

File: src/org/robockets/buttonmanger/buttons/ToggleButton.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+		super(joystickNumber, buttonNumber, command);
+	}
+
+	// Only runs if button is not pushed
+	public void notPressed() {
+		running = false;
+	}
+	
+	public void pressed() {
+		if (!toggle && !running) {
```
Comment (jaredwhitney) [inline src/org/robockets/buttonmanger/buttons/ToggleButton.java]: put inside a if (!running) ?

File: src/org/robockets/buttonmanger/buttons/ToggleButton.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+import org.robockets.buttonmanger.Button;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+public class ToggleButton extends Button {
+	private boolean toggle = false;
+
+	/**
+	 * The command should have some code to run in the run method and other code to run in the interrupted method
```
Comment (jaredwhitney) [inline src/org/robockets/buttonmanger/buttons/ToggleButton.java]: Assuming the run() and interrupted() methods are left to the specific instance of the button?

**Comment by jakekinsella** [comment]
@jaredwhitney When you get a change please review this

**Comment by jaredwhitney** [comment]
:+1:

**Pull request merged**